core/rpc/json/errors.go

```diff
@@ -0,0 +1,23 @@
+package jsonrpc
+
+// General error codes
+const (
+	ErrorUnspecified = iota // don't use
+	ErrorInternal
```

Comment:
Should we comply to SPEC since those are reserved error code?

Reply:
Ah, so for example, use -32601 for `ErrorUnknownMethod`, yeah, should do that.

![image](https://github.com/kwilteam/kwil-db/assets/140431406/82830ff2-7a1b-4463-97a4-7fb8e3dfd648)

That what you had in mind? Seems like a good idea.

Reply:
Yes you're right